prime-angular-frontend/src/app/shared/models/email-for-care-setting.model.ts

```diff
@@ -1,4 +1,5 @@
 export interface EmailsForCareSetting {
   emails: string[];
   careSettingCode: number;
+  healthAuthorityCode: number;
```

Comment:
Should this be `healthAuthorityCode?: number` instead?

Reply:
Good catch. It is updated.